lib/lorekeeper/json_logger.rb

```diff
@@ -87,10 +92,11 @@ def inspect
 
     private
 
-    THREAD_KEY = 'lorekeeper_jsonlogger_key'.freeze # Shared by all threads but unique by thread
-    MESSAGE = 'message'.freeze
-    TIMESTAMP = 'timestamp'.freeze
-    DATE_FORMAT = '%FT%T.%6NZ'.freeze
+    THREAD_KEY = 'lorekeeper_jsonlogger_key' # Shared by all threads but unique by thread
+    LEVEL = 'level'
+    MESSAGE = 'message'
+    TIMESTAMP = 'timestamp'
+    DATE_FORMAT = '%FT%T.%6NZ'
```

Comment:
removed the `.freeze` calls because the magic comment at the top of the file takes care of this.